packages/ra-core/src/actions/listActions.ts

```diff
@@ -71,7 +71,7 @@ export interface ToggleListItemExpandAction {
  * Action creator to toggle the expanded state of a list item
  *
  * @param {string} resource The resource name, e.g. 'posts'
- * @param {string|integer} id The record identifier, e.g. 123
+ * @param {string|int} id The record identifier, e.g. 123
```

Comment:
Where do you see that `int` exists? I don't see it in https://jsdoc.app/tags-param.html#names-types-and-descriptions or https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type

If anything, the `integer` should be changed into the native JavaScript type `Number`, but it carries a lesser constraint. So I'm -1 for this change, in all the files you've edited.

Reply:
Done!